jinahub/indexers/searcher/FaissSearcher/README.md

```diff
@@ -1,63 +1,63 @@
 # FaissSearcher
 
-**FaissSearcher** is a Faiss-powered vector Searcher.
-
-Faiss is a library for efficient similarity search and clustering of dense vectors. It contains algorithms that search in sets of vectors of any size, up to ones that possibly do not fit in RAM. It also contains supporting code for evaluation and parameter tuning. Faiss is written in C++ with complete wrappers for Python/numpy. Some of the most useful algorithms are implemented on the GPU. It is developed by Facebook AI Research.
-
-
+**FaissSearcher** wraps Faiss-powered vector Searcher.
 
+**FaissSearch** is defined as a vector searcher,
```

Comment:
I feel like this is too long. Most Executors only have one or two sentences here. Can you shorten it perhaps?